packages/flutter/lib/src/material/drawer.dart

```diff
@@ -9,6 +9,8 @@ import 'colors.dart';
 import 'list_tile.dart';
 import 'material.dart';
 
+enum DrawerAlignment { START, END }
```

Comment:
s/START/start/
s/END/end/

We use camelCase for enum values.  Also, please add dartdoc for all public identifiers.

Reply:
Fixed